angular.json

```diff
@@ -1,5 +1,9 @@
 {
   "$schema": "./node_modules/@angular/cli/lib/config/schema.json",
+  "cli": {
```

Comment:
This is probably fine, but it probably can also be dropped. Not sure if you have a preference @varnastadeus